sdks/java/core/src/main/java/com/google/cloud/dataflow/sdk/io/AvroIO.java

```diff
@@ -326,6 +327,14 @@
       }
 
       @Override
+      public void populateDisplayData(DisplayData.Builder builder) {
+        builder.add("schema", type);
```

Comment:
Class ends up being a common metadata type, so we support it first-class for display metadata. We'll use the class name (and `shortName`) for UI.

No, we aren't specifying a coder for each transform.

I had assumed the schema information is important-- is that true? If so, is this the best way to capture it?